types/eev/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for eev 0.1
+// Project: https://github.com/chrisdavies/eev
+// Definitions by: boris.stanojevic <https://github.com/borisStanojevic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type CallbackFunction = (data: any) => void;
+
+declare class Eev {
+    constructor();
+    on(names: string, fn: CallbackFunction): void;
+    off(names: string, fn: CallbackFunction): void;
+    emit(name: string, data?: any): void;
+}
+
+declare const eev: Eev;
+
+export = eev;
```

Comment:
Oops, my approval was a mistake. The export is the class `Eev`, not an instance of it. It should be `export = Eev`.

Reply:
Hi Andrew, thanks for doing the review and noticing it.
I have changed that.